makefiles/utils.inc.mk

```diff
@@ -0,0 +1,87 @@
+# Assorted make utilities
+# TODO: move tests out of here
+
+test-all-utils: test-streq test-assert-positive test-assert_eq-positive
+
+# Return an empty string if the arguments are different (works only for single
+# words
+streq = $(filter $(1),$(2))
```

Comment:
This should be more `findstring`, maybe with a `strip` after, to not use patter matching.

Reply:
One requirement for this (I should document it) is that two empty strings are considered different.